### PR TITLE
Add GetPerfStats function

### PR DESCRIPTION
**Pull request by miroslavstastny**
Returns list of perf results on `master` branch, sorted by `build` number desc.

Query parameters:
- `example` - Returns results for all perf examples by default. This param can filter results for a single example
- `buildLt` - Returns last 50 results by default. With this param you can get 50 results before the `build` number

### CODE REVIEW

File: GetPerfStats/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        project.performance = 1;
+    }
+
+    context.log('Query', JSON.stringify(query, null, 2));
+    context.log('Project', JSON.stringify(project, null, 2));
+
+    try {
+        const data = await mongoClient
+            .db('stardust')
+            .collection('stats')
```
Comment (layershifter) [inline GetPerfStats/index.js]: Extract to `config`?

**Comment by layershifter** [comment]
Can you add Prettier and `lint-staged` + `husky`?

File: GetPerfStats/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+/**
+ * Returns list of perf results on master branch, sorted by build number desc
+ * @param example - Returns results for all perf examples by default. This param can filter results for single example.
+ * @param buildLt - Returns last 50 results by default. With this param you can get 50 results before the build number specified.
+ */
+module.exports = async function (context, req) {
+    const mongodbUri = process.env["MONGODB_URI"];
+    const mongodbUser = process.env["MONGODB_USER"];
+    const mongodbPass = process.env["MONGODB_PASS"];
```
Comment (layershifter) [inline GetPerfStats/index.js]: Can be defined outside `module.exports`

**Comment by miroslavstastny** [comment]
> Can you add Prettier and `lint-staged` + `husky`?

#2

File: GetPerfStats/index.js (hunk 1 above)
Comment (miroslavstastny) [reply GetPerfStats/index.js]: Done for `GetPerfStats`, I will do the same for `SaveStats` once I verify it on prod.

**Pull request merged**